packages/account/src/Types/common-prop.type.ts

```diff
@@ -149,3 +149,11 @@ export type TPersonalDetailsForm = {
 } & FormikProps<FormikValues>;
 
 export type TInputFieldValues = Record<string, string>;
+
+export type TAddressDetailFormProps = {
```

Comment:
is it really common types? 
as I see we use it only for the component itself and for tests.
maybe we should keep it in the components file?
for using types in tests we can extract them with React.ComponentProps<typeof TAddressDetailFormProps>

Reply:
I'm using in spec file to type the FormikValue interface